README.md

```diff
@@ -22,50 +22,118 @@ although basic usage is shown below.
 
 ### User Registration Flow
 
+##### Server endpoint (registration challenge):
+
 ```javascript
-// 1. Check that the user is logged in.
+const u2f = require('u2f');
 
-// 2. (On server) generate user registration request and save it in session:
-var u2f = require('u2f');
+function handler(req, res) {
```

Comment:
Good idea - I've combined the two handlers into the same code block, naming one `registrationChallengeHandler` and the other `registrationVerificationHandler`.